core/templates/components/state-editor/state-responses-editor/state-responses.directive.ts

```diff
@@ -138,8 +138,6 @@ angular.module('oppia').directive('stateResponses', [
             if (StateEditorService.isInQuestionMode()) {
               return;
             }
-            var explorationId = ContextService.getExplorationId();
```

Comment:
Maybe we can remove ContextService from this directive too? It does not seem to be used anywhere.